.circleci/config.yml

```diff
@@ -567,7 +552,6 @@ jobs:
       - <<: *recordNonzeroExitCodeIfTestFailed
       - run:
           name: markJobFinishesOnGCS
-          when: always
```

Comment:
don't you need this? the non zero exit code is recorded.

Reply:
I think so - re-added.